fastplotlib/widgets/image.py

```diff
@@ -466,91 +464,56 @@ def __init__(
             for array, order in zip(self.data, self.dims_order):
                 self._dims_max_bounds[_dim] = min(self._dims_max_bounds[_dim], array.shape[order.index(_dim)])
 
-        if self._plot_type == "single":
-            self._plot: Plot = Plot()
+        if grid_plot_kwargs is None:
+            grid_plot_kwargs = {"controllers": "sync"}
+
+        self._plot: GridPlot = GridPlot(shape=grid_shape, **grid_plot_kwargs)
 
-            minmax = quick_min_max(self.data[0])
+        self.image_graphics = list()
+        for data_ix, (d, subplot) in enumerate(zip(self.data, self.plot)):
+            minmax = quick_min_max(self.data[data_ix])
+
+            if self._names is not None:
+                name = self._names[data_ix]
+                name_slider = name
+            else:
+                name = None
+                name_slider = ""
 
             if vmin_vmax_sliders:
                 data_range = np.ptp(minmax)
-                data_range_40p = np.ptp(minmax) * 0.3
+                data_range_40p = np.ptp(minmax) * 0.4
 
                 minmax_slider = FloatRangeSlider(
                     value=minmax,
                     min=minmax[0] - data_range_40p,
                     max=minmax[1] + data_range_40p,
                     step=data_range / 150,
-                    description=f"min-max",
+                    description=f"mm: {name_slider}",
```

Comment:
I initially thought of doing this but the issue is that ipywidget descriptions are very short, if you can find a way to have longer descriptions that would be ideal.

Anyways this will become irrelevent once we make #166